packages/ui-extensions/src/surfaces/admin/components/AdminAction.d.ts

```diff
@@ -102,11 +102,14 @@ declare module 'preact' {
     interface IntrinsicElements {
       [tagName]: Omit<
         HTMLAttributes<HTMLElement>,
-        Extract<keyof HTMLAttributes<HTMLElement>, `on${Capitalize<string>}`>
+        | Extract<keyof HTMLAttributes<HTMLElement>, `on${Capitalize<string>}`>
+        | 'primaryAction'
+        | 'secondaryActions'
       > &
-        ReactProps;
+        AdminActionJSXProps;
```

Comment:
It looks like even though we `Omit` the props above, they are being added back here when using the intersection  operator.